defaults/main.yml

```diff
@@ -1,2 +1,3 @@
 ---
 zeromq_version: 4.0.5
+zeromq_download_url: https://archive.org/download/zeromq_{{ zeromq_version }}/zeromq-{{ zeromq_version }}.tar.gz
```

Comment:
Will this trigger a deprecation warning for "bare variables"? Better add quotes around it.